scripts/typescript_checks_test.py

```diff
@@ -109,17 +109,23 @@ def mock_popen_for_deletion(
             if not os.path.exists(os.path.dirname(MOCK_COMPILED_JS_DIR)):
                 os.mkdir(os.path.dirname(MOCK_COMPILED_JS_DIR))
             typescript_checks.compile_and_check_typescript(
-                typescript_checks.TSCONFIG_FILEPATH)
+                typescript_checks.STRICT_TSCONFIG_FILEPATH)
             self.assertFalse(
                 os.path.exists(os.path.dirname(MOCK_COMPILED_JS_DIR)))
 
-    def test_no_error_is_produced_for_valid_compilation(self) -> None:
+    def test_no_error_is_produced_for_valid_compilation_of_tsconfig_file(self) -> None: # pylint: disable=line-too-long
```

Comment:
```suggestion
    def test_no_error_for_valid_compilation_of_tsconfig(self) -> None: # pylint: disable=line-too-long
```

I think you can shorten the test name to avoid disabling the pylint line length error

Reply:
Done